lib/docs/filters/relay/clean_html.rb

```diff
@@ -2,58 +2,42 @@ module Docs
   class Relay
     class CleanHtmlFilter < Filter
       def call
-        @doc = at_css('.inner-content, article.withtoc')
 
-        if root_page?
-          at_css('h1').content = 'Relay Documentation'
-        end
+        if slug == 'index'
+          css('img').remove
 
-        css('.docs-prevnext', '.hash-link', '.edit-page-link', '.edit-github', 'a.hash', '.edit-page-block', 'a.show', 'a.hide', 'hr').remove
+          css('.projectTitle').each do |node|
+            node.name = 'h1'
+            node.content = 'Relay'
+          end
 
-        css('table h1', 'table h2', 'table h3').each do |node|
-          table = node
-          table = table.parent until table.name == 'table'
-          table.replace(node)
-        end
+          css('pre').remove
 
-        css('a.anchor', 'a.hashref').each do |node|
-          node.parent['id'] ||= node['name'] || node['id']
         end
 
-        css('.highlight').each do |node|
-          node.name = 'pre'
-          node.css('.gutter').remove
-          node['data-language'] = node.at_css('[data-lang]').try(:[], 'data-lang') || 'js'
-          node.content = node.content.strip
-        end
+        css('.docLastUpdate').remove
 
-        css('table.highlighttable').each do |node|
-          node.replace(node.at_css('pre.highlight'))
-        end
+        css('.docs-prevnext').remove
 
-        css('.prism').each do |node|
-          node.name = 'pre'
-          node['data-language'] = node['class'][/(?<=language\-)(\w+)/]
-          node.content = node.content
-        end
+        css('.edit-page-link').remove
 
-        css('blockquote > p:first-child').each do |node|
-          node.remove if node.content.strip == 'Note:'
+        css('h2, h3').each do |node|
+          node.css('a').remove
+          node['id'] = node.content.gsub(/\s/, '-').downcase
```

Comment:
As a general rule, try to use `id`s or `name`s from existing anchors/tags rather than creating new ones. Documents usually have existing links and this can interfere/break them.